### PR TITLE
fix(twig): normalize twig templates, pt6 - INNO-1019

### PR DESCRIPTION
**Pull request by yhuard**
Updated:
- component-label
- component-lang-select-page
- component-lang-select-site
- component-language-list

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 0bc773237124d29c4ee63b572a0294b23a8f3d22

https://deploy-preview-658--europa-component-library.netlify.com

File: src/generic/generic-component-language-list/generic-component-language-list.twig [hunk 1]
```diff
@@ -39,19 +66,19 @@
...
                   {% endif %}
                   {% include '@ecl/generic-component-button' with {
                     'type': 'link',
-                    'label': lang.label ~ (lang.isActive ? '<span class="ecl-icon ecl-icon--check ecl-u-f-r"></span>'),
-                    'href': lang.href,
+                    'label': _lang.label ~ (_lang.isActive ? '<span class="ecl-icon ecl-icon--check ecl-u-f-r"></span>'),
+                    'href': _lang.href,
                     'is_block': true,
-                    'extra_classes': 'ecl-language-list__button' ~ (lang.isActive ? ' ecl-language-list__button--active'),
+                    'extra_classes': 'ecl-language-list__button' ~ (_lang.isActive ? ' ecl-language-list__button--active'),
```
Comment (kalinchernev) [inline src/generic/generic-component-language-list/generic-component-language-list.twig]: I think `isActive` would be `is_active` to match the rest.

**Review by kalinchernev — COMMENTED** [review]
Only 1 comment, which is not a blocker

File: src/generic/generic-component-language-list/generic-component-language-list.twig (hunk 1 above)
Comment (yhuard) [reply src/generic/generic-component-language-list/generic-component-language-list.twig]: you're absolutely right 👍

**Review by kalinchernev — APPROVED** [review]

**Pull request merged**